examples/observability-stack/docker-compose.tracetest.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3.7"
+
+services:
+  tracetest:
+    image: kubeshop/tracetest-agent:latest
+    platform: linux/amd64
+    depends_on:
+      otel-collector:
+        condition: service_started
+    environment:
+      TRACETEST_API_KEY: ${TRACETEST_API_KEY}
```

Comment:
We the agent will access `http://host.docker.internal:8080/`, it's better to add the extra_hosts parameter to make sure Linux users will be able to run this example:

```yaml
extra_hosts:
      - "host.docker.internal:host-gateway"
```